pages/_app.js

```diff
@@ -1,10 +1,28 @@
+import useLocalStorageState from "use-local-storage-state";
 import GlobalStyle from "../styles";
+import getCurrentTimeAndDate from "@/utils/getCurrentTimeAndDate";
+import { uid } from "uid";
 
 export default function App({ Component, pageProps }) {
+  const [tensionEntry, setTensionEntry] = useLocalStorageState("tensionEntry", {
+    defaultValue: [],
+  });
```

Comment:
⚠️ Using a plural name for variables that hold arrays can improve readability. 
**Please also remember to adapt the function names**

```suggestion
const [tensionEntries, setTensionEntries] = useLocalStorageState("tensionEntries", {
  defaultValue: [],
});
  });
```

Please do Adjusting Function Names